rtsp_simulator_ffmpeg_v2.py

```
@@ -41,26 +43,57 @@
+
+# 로깅 설정
+
+# 환경변수 헬퍼 함수
+def get_env_value(key: str, default_value, value_type=str):
+    """환경변수에서 값을 가져오는 헬퍼 함수"""
+    try:
+        value = os.getenv(key)
+        if value is None:
+            return default_value
+        
+        if value_type == bool:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)